### PR TITLE
fix fluent bit error and add index-pattern for systemd

### PR DESCRIPTION
**Pull request by nowjean**

### CODE REVIEW

File: charts/fluent-bit/templates/fluent-bit.conf.tpl [hunk 1]
```diff
@@ -11,12 +11,7 @@
     Name            systemd
     Tag             host.*
     Path            /var/log/journal
-    Mem_Buf_Limit 5MB
-
-[INPUT]
-    Name            systemd
-    Tag             host.*
-    Path            /run/log/journal
```
Comment (mikeln) [inline charts/fluent-bit/templates/fluent-bit.conf.tpl]: We still need this input.  /run/log/journal and /var/log/journal.
True, we may get an error for a missing /run/log/journal...but not always.

**Comment by DStorck** [comment]
can i ask why we're not just using `*` for the es output ?

**Comment by nowjean** [comment]
@DStorck that because make other index-pattern for systemd

File: charts/fluent-bit/templates/fluent-bit.conf.tpl [hunk 2]
```diff
@@ -11,12 +11,7 @@
...
+    Systemd_Filter  _SYSTEMD_UNIT=docker.service
```
Comment (DStorck) [inline charts/fluent-bit/templates/fluent-bit.conf.tpl]: i've also seen a ton of examples of folks using `Systemd_Filter  _SYSTEMD_UNIT=kubelet.service` , not sure which one we want

File: charts/fluent-bit/templates/fluent-bit.conf.tpl [hunk 3]
```diff
@@ -11,13 +11,15 @@
     Name            systemd
     Tag             host.*
     Path            /var/log/journal
+    Systemd_Filter  _SYSTEMD_UNIT=docker.service
```
Comment (DStorck) [inline charts/fluent-bit/templates/fluent-bit.conf.tpl]: i've also seen a ton of examples of folks using Systemd_Filter _SYSTEMD_UNIT=kubelet.service , not sure which one we want

Comment (nowjean) [reply charts/fluent-bit/templates/fluent-bit.conf.tpl]: _SYSTEMD_UNIT=kubelet.service==>  i think it doesn't working my enviroment.

**Comment by nowjean** [comment]
and it makes warning msg like this. " [ warn] [filter_kube] invalid pattern for given tag host.docker.service " 

i'm not sure but,i think this filter only for kubernetes.logs not systemd..

**Pull request closed without merging**